README.md

```diff
@@ -393,10 +393,11 @@ Alternative to the `head` command.
 head() {
     # Usage: head "n" "file"
     while read -r line; do
-        [ "$i" = "$1" ] && break
         printf '%s\n' "$line"
         i=$((i+1))
+        [ "$i" = "$1" ] && return
```

Comment:
I assume this works identically to before?

Reply:
Other than the fact that it doesn't read an extra line, yes, as far as I can tell.